### ISSUE TITLE
[T23][Accessibility][Functionl]Unable to navigate in family tab 

### ISSUE THREAD

**Issue by Maltesh27** [Mohanme13/Feb-19-Bug-bash#202]
Steps to reproduce:
1. Goto Histories
2. Navigate to Family tab 
3. Try to navigate to Qucklist section

Issue: Unable to navigate in family tab
